gui/freehand.py

```diff
@@ -359,6 +363,8 @@ def motion_notify_cb(self, tdw, event, fakepressure=None):
         pressure = event.get_axis(Gdk.AxisUse.PRESSURE)
         xtilt = event.get_axis(Gdk.AxisUse.XTILT)
         ytilt = event.get_axis(Gdk.AxisUse.YTILT)
+        viewzoom = tdw.scale
+        viewrotation = tdw.rotation
```

Comment:
I see what you mean; it would be better to let the brush engine take care as much as it can with the data it is given, right?  One thought I had is that if we ever want a virtual pen rendered along with dab angle (like some other programs do) we'd have to keep (or bring over more) stuff into the gui, duplicating the calculations?

Also, I wonder if other programs interested in the brush engine already have pre-calculated data that would be need to be converted back to "raw"?  I could imagine that being really annoying.

Would it make sense to somehow allow raw data OR calibrated data?  So the brush engine would just use either one.  I don't know how you could do that, but it'd be nice if you could name the data that you fed to the engine "raw_xtilt" vs "calibrated_xtilt", etc.  Then the engine could just have an if statement to handle them differently?